runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler.cpp

```diff
@@ -157,7 +160,12 @@ OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler_initCifNativeTh
 		goto freeAllMemoryThenExit;
 	}
 
-	if (FFI_OK != ffi_prep_cif(cif, FFI_DEFAULT_ABI, argTypesCount, returnType, &(argTypes[0]))) {
+	/* The variadic argument index is -1 by default if it doesn't exist in the argument list.
+	 * Note: it is literally equal to the count of the fixed arguments before variadic arguments.
+	 */
+	status = (varArgIndex < 0) ? ffi_prep_cif(cif, FFI_DEFAULT_ABI, argTypesCount, returnType, &(argTypes[0]))
```

Comment:
This would read a lot better as if/else rather than ?.

Reply:
Agreed and replaced with the if/else statements.